src/cheri_pte.sail

```diff
@@ -15,12 +15,29 @@ bitfield PTE_Bits : pteAttribs = {
 
 /* Reserved PTE bits could be used by extensions on RV64.  There are
  * no such available bits on RV32, so these bits will be zeros on RV32.
+ *
+ * XXX Because these bits are 0 on RV32, do we want to use inverted polarity,
+ * so that 0 is the *enabled* form?
```

Comment:
That's what we also do on MIPS, and for both these reasons I was earlier saying the same thing to Peter.